### DISCUSSION TITLE
[@types/gtag.js] Missing 'ad_user_data' and 'ad_personalization' in  'consent'

### DISCUSSION THREAD

**Discussion by varadarajan-tw** [DefinitelyTyped/DefinitelyTyped#68501] [Issues with a @types package]
Google recently added some [advanced consent features ](https://developers.google.com/tag-platform/security/guides/consent#upgrade-consent-v2) which introduces two new fields `ad_user_data` and `ad_personalization`. The two fields should be updated [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9bbff85ffe3d5f31394e324e3cb4cb57a4d37599/types/gtag.js/index.d.ts#L156).
cc: @AnkitSegment

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Added in PR #68504
